Formula/pulumi.rb

```diff
@@ -16,6 +16,26 @@ class Pulumi < Formula
 
   depends_on "go" => :build
 
+  resource "dill" do
+    url "https://files.pythonhosted.org/packages/e2/96/518a8ea959a734b70d2e95fef98bcbfdc7adad1c1e5f5dd9148c835205a5/dill-0.3.2.zip"
+    sha256 "6e12da0d8e49c220e8d6e97ee8882002e624f1160289ce85ec2cc0a5246b3a2e"
+  end
+
+  resource "grpcio" do
+    url "https://files.pythonhosted.org/packages/e3/0e/f56aa1f8200ae3c5d38305e69f5920caa480c7ff54ae4d8a5f57d1d69fa4/grpcio-1.31.0.tar.gz"
+    sha256 "5043440c45c0a031f387e7f48527541c65d672005fb24cf18ef6857483557d39"
+  end
+
+  resource "protobuf" do
+    url "https://files.pythonhosted.org/packages/25/ef/8f36bc7e4c3dcad241a9e7a51520d30712a7abc034343bed841ff4ea3289/protobuf-3.13.0.tar.gz"
+    sha256 "6a82e0c8bb2bf58f606040cc5814e07715b2094caeba281e2e7d0b0e2e397db5"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/6b/34/415834bfdafca3c5f451532e8a8d9ba89a21c9743a0c59fbd0205c7f9426/six-1.15.0.tar.gz"
+    sha256 "30639c035cdb23534cd4aa2dd52c3bf48f06e5f4a941509c8bafd8ce11080259"
+  end
```

Comment:
these are python dependencies, right?

I think you need specify `depends_on "python@3.8"`?

Reply:
Hi @chenrui333 

So the bump-formula-pr tool added these by default - so I'd love some guidance here on what you need

Paul

Reply:
ok, maybe we should remove these dependencies. cc @dtrodrigues